phd-tests/tests/src/migrate.rs

```diff
@@ -84,6 +85,147 @@ mod from_base {
     }
 }
 
+/// Tests for migrations while a process is running on the guest.
+mod running_process {
+    use super::*;
+
+    #[phd_testcase]
+    fn migrate_running_process(ctx: &Framework) {
+        let mut source =
+            ctx.spawn_default_vm("migrate_running_process_source")?;
+        let mut target = ctx.spawn_successor_vm(
+            "migrate_running_process_target",
+            &source,
+            None,
+        )?;
+
+        source.launch()?;
+        source.wait_to_boot()?;
+
+        mk_dirt(&source)?;
+
+        target.migrate_from(
+            &source,
+            Uuid::new_v4(),
+            Duration::from_secs(60),
```

Comment:
I just landed #627, so this can use `MigrationTimeout::default` after a merge/rebase.